docs/pages/blog/v6-beta-pickers.md

```diff
@@ -58,18 +58,27 @@ Not only can you type the date as text, but you can also use arrow keys to navig
   <source src="/static/blog/v6-beta-pickers/date-field-navigation.mp4" type="video/mp4" />
 </video>
 
-Moreover, notice in the following example that as we increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+For a quick comparison, let's check out how the old and new approaches behave in typical scenarios.
 
-<video style="margin-bottom: 5px; width: 608px;" autoplay muted loop playsinline>
-  <source src="/static/blog/v6-beta-pickers/smart-field.mp4" type="video/mp4" />
+<video style="margin-bottom:24px" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/quick-comparison-fields.mp4" type="video/mp4" />
 </video>
 
-For a quick comparison, let's check out how the old and new approaches behave in typical scenarios.
+Moreover, notice in the following example that as we edit the day, the component automatically navigates only in valid values.
 
-<video style="margin-bottom:24px" autoplay muted loop playsinline>
-  <source src="/static/blog/v6-beta-pickers/quick-comparison-fields.mp4" type="video/mp4" />
+<video style="margin-bottom: 5px; width: 608px;" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/smart-field-stable.mp4" type="video/mp4" />
 </video>
 
+:::info
+**Behavior change alert**
+
+During pre-releases, using <kbd>Arrow Up</kbd> and <kbd>Arrow Down</kbd> to update a date section would essentially update the entire field like you were navigating the calendar.
+In the previous example, updating the day value from 28 to 1 would also update the month to March.
+
+On the stable version, released after the original post, this behavior [has been changed](https://github.com/mui/mui-x/issues/7934), and each date section became independent.
```

Comment:
Should we explain briefly why ? So that people don't see it as a regression ?

Reply:
Updated